src/editor-finder.ts

```diff
@@ -93,11 +95,13 @@ export class EditorFinder implements IDisposable, IDebuggerEditorFinder {
    * @param debugSessionPath The path for the current debug session.
    * @param source The source to find.
    * @param focus - Set to true to focus on the relevant cell. Default to false.
+   * @param kernelName The kernel name from current session.
    */
   private _findInNotebooks(
     debugSessionPath: string,
     source: string,
-    focus: boolean
+    focus: boolean,
+    kernelName: string
```

Comment:
Similar comment as above. These `findIn*` methods used to have 2 arguments only, but now that they are growing we might want to refactor that into an object too.